src/styles/blocks/day.scss

```diff
@@ -0,0 +1,25 @@
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $cell-size;
+  height: $cell-size;
+
+  border: 1px solid black;
+  background-color: #eee;
+
+  transition: background-color, transform 0.5s;
+
+  &:hover {
+    cursor: pointer;
```

Comment:
Do you need cursor pointer only when hover

Reply:
On hovering over a cell, the cursor should become pointer.(rules from task)